frontend/app/tests/e2e/pages/history-page/index.ts

```diff
@@ -5,6 +5,7 @@ export class HistoryPage {
     cy.get('.navigation__history').click();
     cy.get('.navigation__history-trades').scrollIntoView();
     cy.get('.navigation__history-trades').should('be.visible');
+    cy.get('.navigation__history-trades').trigger('mouseover');
```

Comment:
So I want to fix the flakiness here that happen often,
What I see from the video is, the `button` is not being clicked here.

I look into https://stackoverflow.com/questions/51254946/cypress-does-not-always-executes-click-on-element and https://github.com/cypress-io/cypress/issues/2928

It seems this line helps.
Maybe we can consider to move to `playwright` after the release, cypress is too bad.
Also I read somewhere https://www.reddit.com/r/QualityAssurance/comments/yqhg55/cypress_vs_playwright_some_comparison_elements/ that cypress will be soon become paid tool.